crates/jxl-oxide/src/lib.rs

```diff
@@ -106,9 +107,18 @@ pub use jxl_image::{ExtraChannelType, ImageHeader};
 use jxl_render::{RenderContext, IndexedFrame};
 
 pub use fb::FrameBuffer;
+pub use jxl_threadpool::JxlThreadPool;
 
 pub type Result<T> = std::result::Result<T, Box<dyn std::error::Error + Send + Sync + 'static>>;
 
+fn default_pool() -> JxlThreadPool {
+    if cfg!(feature = "rayon") {
+        JxlThreadPool::default()
+    } else {
+        JxlThreadPool::none()
+    }
+}
```

Comment:
`Default` for `JxlThreadPool` is implemented for for both `#[cfg(feature = "rayon")]` and `#[cfg(not(feature = "rayon"))]` cases

Reply:
This is intended. I've wrote some explanation in the corresponding commit:

> Cargo features are additive, so thread pool support might be enabled
> even if it's disabled in jxl-oxide crate.

Well maybe I'm going to copy that commit message there...